src/channels-api.js

```diff
@@ -386,23 +386,23 @@ class ChannelsApi implements IStreamingAPI {
     }
   }
 
-  request(stream: string, payload: Object, requestId: string=UUID.generate()): Promise<Object> {
+  request(stream: string, payload: Object, request_id: string=UUID.generate()): Promise<Object> {
     return new Promise((resolve, reject) => {
-      const selector = this._buildRequestResponseSelector(stream, requestId);
+      const selector = this._buildRequestResponseSelector(stream, request_id);
 
       this.dispatcher.once(selector, data => {
         const {payload: response} = data;
-        const {responseStatus} = response;
+        const {response_status} = response;
```

Comment:
Same idea here. Would you forego destructuring assignment, and use `const responseStatus = response.response_status`?